packages/atomic/scripts/svg-transform.mjs

```diff
@@ -0,0 +1,66 @@
+import {readFileSync} from 'fs';
+import {basename, dirname, join, resolve} from 'path';
+import {
+  NodeFlags,
+  isImportDeclaration,
+  visitEachChild,
+  visitNode,
+} from 'typescript';
+
+function createStatement(factory, svgContent, variableName) {
+  const bindingName = undefined;
+  const exclamationToken = undefined;
+  const modifiers = [];
+  const {
+    createVariableStatement,
+    createVariableDeclarationList,
+    createVariableDeclaration,
+    createStringLiteral,
+  } = factory;
+
+  if (variableName === undefined) {
+    throw new Error(
+      `Variable name is not defined for the import statement ${node.getText()}`
+    );
+  }
+
+  return createVariableStatement(
+    modifiers,
+    createVariableDeclarationList(
+      [
+        createVariableDeclaration(
+          variableName,
+          bindingName,
+          exclamationToken,
+          createStringLiteral(svgContent)
```

Comment:
converting the SVG content into a string literal for the right hand side of the assignation